### DISCUSSION TITLE
Can't run Kryptor on Windows 10

### DISCUSSION THREAD

**Discussion by neverwhere40** [samuel-lucas6/Kryptor#25] [Q&A] [answered]
I am using the 3.0.2 Beta in Windows 10 and the program will not load. When I extract all the files to their own directory and then click on the kryptor.exe file, a window flashes for a second, but the program does not load. I have tried to run the program as Administrator and this does not change the behavior. I has also temporarily disabled Avast and this does not affect the behavior of the program launching. The version that does work on my Windows 10 machine is 2.2,2 Beta, which is an older version of the program.

**Comment by samuel-lucas6** [comment, marked as answer]
Hi @neverwhere40,

Kryptor has switched from a GUI application to a CLI application, meaning you now need to run it from the Command Prompt. For instructions, please refer to this [page](https://www.kryptor.co.uk/installation#running-kryptor-on-windows) of the documentation. I hope that helps.
